modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java

```diff
@@ -967,7 +979,7 @@ private String _normalizeFriendlyURL(String friendlyURL) {
 
 	private Locale _setAlternativeLayoutFriendlyURL(
 		HttpServletRequest httpServletRequest, Layout layout,
-		String friendlyURL, SiteFriendlyURL siteFriendlyURL) {
+		String friendlyURL, SiteFriendlyURL siteFriendlyURL, long companyId) {
```

Comment:
Can you put the companyId as the first argument?

Reply:
Done